doc/user/routemap.rst

```diff
@@ -7,6 +7,9 @@ Route Maps
 Route maps provide a means to both filter and/or apply actions to route, hence
 allowing policy to be applied to routes.
 
+For a route reflector to apply ``route-map``s to reflected routes, be sure to include
```

Comment:
``route-map``s is formated not very correctly. 

You can always double-check what you did in "view file" mode.

![Screenshot from 2019-12-01 18-08-25](https://user-images.githubusercontent.com/3352707/69916597-a292bf00-1465-11ea-9f3b-c1e957469bd4.png)

By the way, please squash to a single commit and use `git commit --amend -s` (to sign-off).